migrate/20250501_discount_code.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:discount_code) do
+      column :id, :uuid, primary_key: true
+      column :created_at, :timestamptz, null: false, default: Sequel::CURRENT_TIMESTAMP
+      column :discount_code, :text, null: false, unique: true
```

Comment:
It allows for case-insensitive text filtering.

```suggestion
      column :discount_code, :citext, null: false, unique: true
```

Reply:
Done.